internal/protoplugin/protoplugin.go

```diff
@@ -159,6 +159,16 @@ type Service struct {
 	Methods []*Method
 }
 
+// FQSN returns a fully qualified service name of this service.
```

Comment:
It's the difference between `KeyValue` and `.uber.yarpc.internal.examples.protobuf.example.Keyvalue` - protobuf has a `.` at the beginning for some reason, this is standard (and very annoying).